customize-example/Dockerfile.redhat-ubi-jfrog-rabbitmq

```diff
@@ -2,7 +2,7 @@
 # https://github.com/bitnami/bitnami-docker-rabbitmq/blob/master/3.8/debian-10/Dockerfile
 # JFROG REDHAT UBI PORT TO WORK IN BITNAMI RABBITMQ HELM CHARTS
 
-FROM bitnami/rabbitmq:3.8.3-debian-10-r40 as base
+FROM bitnami/rabbitmq:3.8.9-debian-10-r20 as base
```

Comment:
@peters95  can we pull images from `releases-docker.jfrog.io` instead of bitnami ?

Reply:
I want to multi stage off the official bitnami image for Redhat UBI8. Does releases contain the same official bitnami image?

Reply:
We can leave it for now, will discuss this internally